xtraplatform-features/src/test/groovy/de/ii/xtraplatform/features/domain/FeatureProviderDataV2Spec.groovy

```diff
@@ -49,18 +55,88 @@ class FeatureProviderDataV2Spec extends Specification {
+                                .type(FeaturePropertyV2.Type.OBJECT_ARRAY)
+                                .objectType("Link")
+                                .label("Versuche")
+                                .constraints(ImmutableMap.of("minOccurrence", ImmutableList.of("1"), "maxOccurrence", ImmutableList.of("3")))
+                                .putProperties2("title", new ImmutableFeaturePropertyV2.Builder()
+                                        .path("category_fk")
+                                )
+                                .putProperties2("href", new ImmutableFeaturePropertyV2.Builder()
+                                        .path("id")
+                                        .constraints(ImmutableMap.of("required", ImmutableList.of("true")))
+                                )
+                        )
+                        .putProperties2("filterValues", new ImmutableFeaturePropertyV2.Builder()
+                                .path("[id=observationsubject_fk]observationsubject_filtervalues")
+                                .type(FeaturePropertyV2.Type.OBJECT_ARRAY)
+                                .objectType("FilterValue")
+                                .label("Leitkennwerte")
+                                .putProperties2("property", new ImmutableFeaturePropertyV2.Builder()
+                                        .path("filtervalueproperty_fk")
+                                        .label("Eigenschaft")
+                                )
+                                .putProperties2("property", new ImmutableFeaturePropertyV2.Builder()
+                                        .path("filtervalueproperty_fk")
+                                        .label("Eigenschaft")
+                                )
+                                .putProperties2("measure", new ImmutableFeaturePropertyV2.Builder()
+                                        .path("resultmeasure")
+                                        .type(FeaturePropertyV2.Type.FLOAT)
+                                        .label("Messwert")
+                                        .constraints(ImmutableMap.of("min", ImmutableList.of("0.0"), "max", ImmutableList.of("100.0")))
+                                )
+                                .putProperties2("classification", new ImmutableFeaturePropertyV2.Builder()
+                                        .path("[id=filtervalue_fk]filtervalue_resultclassification/classificationcode_fk")
+                                        .type(FeaturePropertyV2.Type.VALUE_ARRAY)
+                                        .valueType(FeaturePropertyV2.Type.STRING)
+                                        .label("Klassifikationen")
+                                )
                         )
                 )
+                .putCodelists("nullValues", ImmutableMap.of("No Information", "null", "-1", "null"))
```

Comment:
If the -1 is serialized without quotes, that's ok. Just apply the change to `geoval.yml`.